zasilkovna.php

```diff
@@ -813,6 +825,63 @@ public function plgVmDisplayListFEShipment(VirtueMartCart $cart, $selected = 0,
         return TRUE;
     }
 
+    /**
+     * @param ShipmentMethod $shipmentMethod
+     * @param string $countryCode
+     * @param array<int, int|string> $carrierIds
+     * @return array
+     */
+    private static function createWidgetVendorsParam(
+        ShipmentMethod $shipmentMethod,
+        $countryCode,
+        array $carrierIds = []
+    ) {
+        $widgetVendors = [];
+        $vendorGroups = $shipmentMethod->getVendorGroups();
+
+        foreach ($vendorGroups as $vendorGroup) {
+            if (!VendorGroups::isGroupPresentInCountry($vendorGroup, $countryCode)) {
+                continue;
+            }
+
+            $widgetVendors[] = self::createWidgetVendor($countryCode, $vendorGroup, null);
+        }
+
+        if (empty($widgetVendors)) {
+            return $widgetVendors;
```

Comment:
Takže pro země bez interních VM chceš vždy prázdné pole?

Reply:
Ano. Prázdné pole způsobí, že se zobrazí staré rozhraní widgetu, protože se nepředá parametr vendors. Widget v takovém případě zobrazuje zákazníkovi všechny možnosti.

Jaroslav to chce mít jednoduché bez migrací a jen se třemi checkboxy. Takže pro nenavolené/nevalidní typy interních výdejních míst se zobrazuje původní widget.

A carrierId se předává pouze do nového rozhraní, protože původní řešení to nepředávalo vůbec.